### ISSUE TITLE
only metadata

### ISSUE THREAD

**Issue by Heerpa** [tlambert03/nd2#71]
* nd2 version: 0.2.2
* Python version: 3.7.13
* Operating System: Windows Server 2012

### Description

Is there a way to open a file only for reading metadata, without the overhead generated by preparing to read image data?

**Comment by Heerpa** [comment]
This basically how I do it, just without the context manager:

```
import nd2

f = nd2.ND2File('some_file.nd2')
f.attributes        # nd2.structures.Attributes
f.metadata          # nd2.structures.Metadata
f.experiment        # List[nd2.structures.ExpLoop]
f.text_info         # dict of misc info
f.custom_data       # mishmash of data extracted from file
f.close()
```
Opening the (large) file takes a few seconds, so I was assuming that image data is being prepared there already. Happy to close the issue if that's not the case.

**Comment by tlambert03** [comment]
Oh I just noticed you’re on 0.2.2, can you update and try again?

**Comment by Heerpa** [comment]
Yes, with 0.2.5, opening the file takes a fraction of a second instead of multiple seconds. Thanks very much!

**Comment by tlambert03** [comment]
excellent!  you might also try, with v0.3.0:

```python
import nd2

with nd2.ND2File(..., read_using_sdk=True):
    # get your metadata
```

I'd be curious to hear what you find there regarding performance.

Backstory:
This library actually contains _both_ a wrapper around the official nd2 sdk (which is why all the metadata comes out so nice), but also a direct chunkmap-inspection and numpy-memmap reader.  The latter was introduced when I found what seemed to be performance benefits of not using the sdk for the actual reading process.

However, that "greedy" inspection of the chunkmap was what led to your original performance hit observations.  While that was fixed in 0.2.5, the new `read_using_sdk` parameter in v0.3.0 essentially opts out of _all_ non-sdk logic (i.e. it doesn't even open a separate file handle for any additional inspection).  So, you might even see an additional bump for pure-metadata purposes there.  But, if my previous observations are correct, you _might_ see a slight hit on actual data read speed... but I'm not sure :)
